### ISSUE TITLE
Detail implementation

### ISSUE THREAD

**Issue by Depechie** [RolandPheasant/DynamicData.Snippets#6]
Any more details on how we can get a hold of the mainthread in a Xamarin app?
https://github.com/RolandPheasant/DynamicData.Snippets/blob/master/DynamicData.Snippets/Group/XamarinFormsGrouping.cs#L31

Not been able to create a provider.

**Comment by RolandPheasant** [comment]
The provider is used as an example for how to set up observables so they can be tested and is not a definitive means of solving this problem.

To get the main thread ReactiveUI provide an out of the box solution for accessing the main thread. I suggest you check it out.

**Comment by Depechie** [comment]
Yeah I know, problem is I'm not using Reactive UI ;) but will check their code. Thx.
